testing/asserts.ts

```diff
@@ -92,18 +92,15 @@ export function assertArrayIncludes<T>(
  * assertEquals("hello", "world"); // Throws
  * ```
  *
- * Note: formatter option is experimental and may be removed in the future.
- *
  * @deprecated This will be removed in 1.0.0. Import from
  * {@link https://jsr.io/@std/assert | @std/assert} instead.
  */
 export function assertEquals<T>(
   actual: T,
   expected: T,
   msg?: string,
-  options: { formatter?: (value: unknown) => string } = {},
 ) {
-  asserts.assertEquals<T>(actual, expected, msg, options);
```

Comment:
How about leaving this unchanged because this API is already deprecated and removed at 1.0.0-rc.1

Reply:
I think we should keep the APIs between `@std/assert` and `@std/testing/asserts` synchronized, as it makes any migration seamless. Also, keeping this unchanged would mean I'd have to re-implement `assertEquals()` here in its previous state.

Reply:
Ah ok. It depends on `@std/assert`.. I missed it.